build/process.py

```diff
@@ -12,9 +11,20 @@
 import numpy as np
 import warnings
 warnings.simplefilter('ignore')
+import yaml
+import os
 
+def price_change(dataframe,column_1,column_2,new_column):
+    dataframe[new_column] = (dataframe[column_1] - dataframe[column_2])/dataframe[column_2]
+    dataframe.loc[np.isinf(dataframe[new_column]), new_column] = np.nan
+    return dataframe
 
 def clean_add_features():
```

Comment:
functions should generally be at most 20 lines, so you should create subfunctions (rule of thumb is if you can group a few lines together and give that function a meaningful name) and then call them all under clean_add_features